src/types/TextSizeCustomization.ts

```diff
@@ -0,0 +1,31 @@
+type TextSizeCustomization =
```

Comment:
I would just call this `TextSize`

```suggestion
type TextSize =
```